js/create-elements.js

```diff
@@ -23,48 +23,66 @@
     cardElement.querySelector('.popup__text--time').textContent = 'Заезд после ' + cardStructure.offer.checkin + ', выезд до ' + cardStructure.offer.checkout;
     cardElement.querySelector('.popup__features').textContent = cardStructure.offer.features;
     cardElement.querySelector('.popup__description').textContent = cardStructure.offer.description;
-    cardElement.querySelector('.popup__photo').src = cardStructure.offer.photos;
+    for (var i = 0; i < cardStructure.offer.photos.length; i++) {
+      var fragment = document.createDocumentFragment();
+      var imageLayout = cardElement.querySelector('.popup__photo').cloneNode(true);
+      var realImages = fragment.appendChild(imageLayout);
+      realImages.src = cardStructure.offer.photos[i];
+      cardElement.querySelector('.popup__photos').appendChild(realImages);
+    }
+    cardElement.querySelector('.popup__photo').parentNode.removeChild(cardElement.querySelector('.popup__photo'));
     return cardElement;
   };
 
   var createPin = function (pinStructure) {
     var cardPin = similarPinTemplate.cloneNode(true);
-    var allocate = 'left: ' + (pinStructure.location.x - window.utils.PIN_SIZE / 2) + 'px; ' + 'top: ' + (pinStructure.location.y - window.utils.PIN_SIZE) + 'px;';
-    cardPin.style = allocate;
+    cardPin.style = 'left: ' + (pinStructure.location.x - window.utils.PIN_SIZE / 2) + 'px; ' + 'top: ' + (pinStructure.location.y - window.utils.PIN_SIZE) + 'px;';
     cardPin.querySelector('img').src = pinStructure.author.avatar;
     cardPin.querySelector('img').alt = pinStructure.offer.title;
+    cardPin.classList.add('map__pin__advert');
     return cardPin;
   };
 
-  var mapCardHidden = function () {
+  var mapCardHide = function () {
```

Comment:
```suggestion
  var hideCardsOnMap = function () {
```

сначала действие - "прячем", потом что - "карточки (во мн. ч.) на карте"

кстати, это карточки чего?

Reply:
Это карточки объявлений, прячу их чтоб не показывались до нажатия по определенному пину